gotocompany/compass/v1beta1/service.proto

```diff
@@ -1146,6 +1178,58 @@ message DeleteAssetRequest {
 
 message DeleteAssetResponse {}
 
+message DeleteAssetsRequest {
```

Comment:
do we need all the fields here? can we reduce it further if it's not really required at the moment?

Reply:
Currently, my idea is Delete Assets will use Filter type in Compass (same as GetAllAssets), with mandatory filled for duration_for_upper_threshold field.

So, I just make it similar to GetAllAssetsRequest, except unnecessary field for DeleteAssets (e.g, size, sort).

Reply:
Any concern mbak regarding that?

Reply:
Need your approval since you request changes in here 😅 🙏

Reply:
oh sorry mas Anwar, so I thought irainia was another person (woman) 😆